sdk/cosmosdb/cosmos/src/utils/hashing/encoding/number.ts

```diff
@@ -51,7 +51,7 @@ function encodeNumberAsUInt64JSBI(value: number): JSBI {
   const rawValueBits = getRawBitsJSBI(value);
   const mask = JSBI.BigInt(0x8000000000000000);
   const returned =
-    rawValueBits < mask
+    JSBI.toNumber(rawValueBits) < JSBI.toNumber(mask)
```

Comment:
We might lose precision here. is it ok?

Reply:
yes. will correct this. we can use JSBI.greaterThan() instead.